contrib/containerized-pbench/README

```diff
@@ -0,0 +1,8 @@
+mkdir var_lib_pbench-agent
+mkdir fiotest
+vi example.sh && chmod 755 example.sh
+podman run --rm --network host --name example-with-bash -v ./example.sh:/example.sh:z -v ./var_lib_pbench-agent:/var/lib/pbench-agent:z -v ./fiotest:/fiotest:z quay.io/pbench/pbench-agent-all-centos-8:latest /example.sh
+podman run --rm --network host -it -v ./var_lib_pbench-agent:/var/lib/pbench-agent:z quay.io/pbench/pbench-agent-all-centos-8:latest bash
+  - Edit /opt/pbench-agent/config/pbench-agent.cfg to set target hostname
+  - pbench-generate-token
+  - pbench-results-move --token "${token}"
```

Comment:
Line 7 might be better as
```
- token=$(pbench-generate-token)
```

Reply:
Unfortunately, that does not work because the `Username: ` prompt becomes part of the `stdout` collected by `$()`.  We need to change the prompting to always use the console or prompt via `stderr`.

Reply:
I have posted a change to `pbench-generate-token` to add a `--output` option and leveraged that in the script (though it is untested due to a bit of a chicken-and-egg problem).

Reply:
How does `--output` help support prompting to the terminal in the script?

Another approach would be to prompt from the script to get username and password, then pass them into `pbench-generate-token --username <foo> --password <bar>` so we don't need to worry about prompts or redirecting output.

But in any case, with "API keys" right around the corner replacing `pbench-generate-token` entirely, I'm not going to worry about it too much.

Reply:
I was trying to minimize the amount of work it takes.  The `--output` path seemed easiest to make work without getting into a repeat of prompting already handled by `pbench-generate-token`.

As for "API keys" and the like, it is not going to happen in time for the mid-March release date, so I wanted to make a helpful change that will be around for a while.

Reply:
@dbutenhof note that the change `pbench-generate-token` is handled in PR #3264.

Reply:
All we have to do is have generate-token send the token to `stderr` and then use shell redirection to swap `stderr` and `stdout`.  🤣

Reply:
Already made the proposed change in #3264.  I am not a fan of playing games with stderr as "good data" since "standard error" implies errors.

Reply:
> I am not a fan of playing games with stderr

(In my best Foghorn Leghorn voice...) "Uh, I say, that was, uh, a joke, son -- a joke!"  😆 

(As I recall, it's actually pretty hard to swap `stderr` and `stdout` using shell redirection.)